src/components/ChallengeEditor/NDAField/index.js

```diff
@@ -1,26 +1,58 @@
 import _ from 'lodash'
 import React from 'react'
 import PropTypes from 'prop-types'
+import cn from 'classnames'
 import styles from './NDAField.module.scss'
 import { DEFAULT_NDA_UUID } from '../../../config/constants'
 
 const NDAField = ({ challenge, toggleNdaRequire, readOnly }) => {
   const isRequiredNda = challenge.terms && _.some(challenge.terms, { id: DEFAULT_NDA_UUID })
+
+  if (readOnly) {
+    return (
+      <div className={styles.row}>
+        <div className={cn(styles.field, styles.col1, styles.fieldTitle)}>NDA Required :</div>
+        <div className={cn(styles.field, styles.col2)}>
+          {
+            isRequiredNda
+              ? <div>Yes</div>
+              : <div>No</div>
```

Comment:
@CDharmateja I guess we don't need to wrap each answer "yes" / "no" into `<div>` and can simplify this to:
```js
{isRequiredNda ? 'Yes' : 'No'}
```